scripts/longtermeffects.lua

```diff
@@ -61,14 +61,12 @@ function nextRound_new(nRounds, bTimeChanged)
 		-- Announce round
 		nCurrent = nCurrent + 1;
 		
-		-- bmos resetting rounds and advancing time
-		if nCurrent >= 10 and not bTimeChanged then
-			local nMinutes = math.floor(nCurrent / 10)
-			nCurrent = nCurrent - (nMinutes * 10)
-			CalendarManager.adjustMinutes(nMinutes)
+		-- bmos advancing time
+		if (nCurrent % 10) = 9 and not bTimeChanged then
+			CalendarManager.adjustMinutes(1)
 			CalendarManager.outputTime()
 		end
```

Comment:
If we are no longer having side effect of nCurrent changing I would suggest to extract both this fragment and next occurrence into a function.

Reply:
I think it needs to be == here rather than =.